README.md

```diff
@@ -150,6 +150,8 @@ render((
 ), document.getElementById("app"));
 ```
 
+WARNING: The `formData` field is **not** used only for initialization but it sets the data of the form if your wrapping component rerenders. Therefore make sure you listen to the `onChange` event and update the data you pass to the `formData` field.
```

Comment:
I would say: *Therefore, if you have situations where your parent component can re-render, make sure you listen ...* 

Because in our use-cases we never had to do that, and it can be confusing to read that at first. What do you think?